run_development.sh

```diff
@@ -1,15 +1,18 @@
 #!/bin/bash
 
 CURRENT_PATH="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
- 
+
+docker pull ghcr.io/navikt/pdfgen:2.0.30
 docker run \
         -v "$CURRENT_PATH/templates:/app/templates" \
         -v "$CURRENT_PATH/fonts:/app/fonts" \
         -v "$CURRENT_PATH/data:/app/data" \
         -v "$CURRENT_PATH/resources:/app/resources" \
         -p 8080:8080 \
+        -e DEV_MODE=true \
         -e DISABLE_PDF_GET=false \
+        -e JDK_JAVA_OPTIONS \
         -it \
         --rm \
-        syfopdfgen:latest
+        ghcr.io/navikt/pdfgen:2.0.30
```

Comment:
Vil det være fare her for at dependabot oppdaterer kun dockerfile og ikke denne, så de kommer i usync?